src/libleo/leo/leoinquiry.c

```diff
@@ -0,0 +1,23 @@
+#include "global.h"
+#include "ultra64/leo.h"
+#include "ultra64/leoappli.h"
+#include "ultra64/leodrive.h"
+
+void leoInquiry(void) {
+    u32 asic_id;
+    u32 asic_data;
+
+    osEPiReadIo(LEOPiInfo, ASIC_ID_REG, &asic_id);
+    if (leoSend_asic_cmd_w(0x1B0000, 0) == 0) {
```

Comment:
```suggestion
    if (leoSend_asic_cmd_w(0x1B0000, 0) == LEO_ERROR_GOOD) {
```
Maybe

Reply:
not sure if this should be `LEO_SENSE_NO_ADDITIONAL_SENSE_INFOMATION` [sic] actually. I started trying to label all the sense codes but that quickly became a huge rabbit hole, and in general I'm not sure when to use SENSE vs ERROR since they're often interchangeable

Reply:
I think we can leave this for a later cleanup